### DISCUSSION TITLE
How to set indent spaces for round bracket?

### DISCUSSION THREAD

**Discussion by walsripe** [microsoft/vscode-discussions#1032] [Extension Development QnA]
Hi,

I want to modify the indent for function definition after clicking Return in the round bracket like the screenshot below. I want it to work in real-time rather than save and format mode. Is there a good idea about how to set up this function in the VSCODE?

I appreciate your help.

<img width="1198" alt="image" src="https://github.com/microsoft/vscode-discussions/assets/26459647/9ec9b0d2-cf49-468d-8a4e-d6ae7b36e87c">

**Comment by isidorn** [comment]
Just check vscode settings - an indent one should help. Also MS Copilot or ChatGPT can help with this.
